app/admin/practices.rb

```diff
@@ -9,92 +11,28 @@
     id_column
     column :name
     column :support_network_email
+    column(:user) {|practice| practice.user&.email}
```

Comment:
TIL! https://rubyinrails.com/2017/11/17/safe-navigation-operator-ampersand-dot-in-ruby/